test/vcf/record_test.cpp

```diff
@@ -95,12 +95,12 @@ namespace ebi
                                 "chr1", 
                                 123456, 
                                 { "id123", "id456" }, 
-                                "A", 
-                                { "AC", "AT" }, 
+                                vcf::A, 
+                                { vcf::AC, "AT" }, 
```

Comment:
revert back AC to hardcoded string in this file, as it is used as a bases string too.